docs/static/releasenotes.asciidoc

```diff
@@ -55,6 +56,43 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
 
+[[logstash-8-13-0]]
+=== Logstash 8.13.0 Release Notes
+
+* Package `elastic_integration` plugin. https://github.com/elastic/logstash/pull/15769[#15769]
+* Flow metrics added `worker_utilization` to pipeline-level https://github.com/elastic/logstash/pull/15912[#15912]
+
+==== Plugins
+
+*Elastic_integration Filter - 0.1.6*
+
+* Using this filter you can process Elastic integrations powered by Elasticsearch Ingest Node in Logstash.
```

Comment:
This is copied from the plugin description.
"Elastic integrations" is an overloaded term that I am not sure how reader will interpret it.

Reply:
Maybe something like this.

This plugin executes ingest pipeline for each event in Logstash. Once configured to point to an Elasticsearch cluster, this filter will detect which ingest pipeline to execute, using an explicitly-defined `pipeline_name` or auto-detecting the event’s data-stream and its default pipeline.

Logstash loads ingest pipeline definition from Elasticsearch and runs that pipeline inside Logstash without transmitting the event to Elasticsearch. Any downstream Elasticsearch output in the Logstash pipeline will avoid running the event’s default pipeline again in Elasticsearch.

Reply:
We want to _avoid_ mentions of "ingest pipeline", please. 

Ingest Pipeline is the _"how"_, not the "what" or the "why".